common/pupil/token.ts

```diff
@@ -7,6 +7,7 @@ import { userForPupil } from '../user';
 const logger = getLogger('Pupil Token Auth');
 
 export async function refreshToken(pupil: pupil): Promise<string | never> {
+    //PR Note: Do we have to drop this?
     if (pupil.verification !== null) {
```

Comment:
We have to change it to check if `verifiedAt` is not set

Reply:
We still need to change this validation to check if verified is not set @paul-jaworski
Also, remove the comment